mdm/checkin.go

```diff
@@ -10,31 +10,31 @@ import (
 	"github.com/pkg/errors"
 )
 
-func (svc *MDMService) Checkin(ctx context.Context, event CheckinEvent) error {
+func (svc *MDMService) Checkin(ctx context.Context, event CheckinEvent) ([]byte, error) {
```

Comment:
Thoughts on adding a function comment to explain what the returned `[]byte` represents and can be used for?

Reply:
I realize there might be many functions which don't currently adhere to this standard of commenting but I think it would still be of value here. 

https://dave.cheney.net/practical-go/presentations/qcon-china.html#_always_document_public_symbols

Reply:
Sure. I met you half-way. All of the implementors of the interface I add a small comment to. But I also added documentation to the main interface.

Reply:
I walked this back, actually. In the document you posted:

> There is one exception to this rule; you don’t need to document methods that implement an interface. Specifically don’t do this:

So I removed those. :)